Codigo/Modulo_UsUaRiOs.bas

```diff
@@ -3028,3 +3028,81 @@ End Sub
 Public Function IsVisible(ByRef User As t_User)
     IsVisible = Not (User.flags.invisible Or User.flags.Oculto)
 End Function
+
+Public Function CanHelp(ByVal UserIndex As Integer, ByVal TargetUserIndex As Integer) As e_InteractionResult
```

Comment:
I'd suggest a more descriptive name, `CanHelp` does not describe the purpose of the function